tests/manage/mcg/test_mcg_resources_disruptions.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    ManageTest, tier4, tier4a, ignore_leftovers, skipif_ocs_version, on_prem_platform_required
+)
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs import constants, defaults, cluster
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def setup(request):
+    request.cls.cl_obj = cluster.CephCluster()
+
+
+@tier4
+@tier4a
+@ignore_leftovers()
+@pytest.mark.usefixtures(setup.__name__)
+class TestMCGResourcesDisruptions(ManageTest):
+    """
+    Test MCG resources disruptions
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["resource_to_delete"],
+        argvalues=[
+            pytest.param(
+                *['noobaa_core'], marks=pytest.mark.polarion_id("OCS-2232")
+            ),
+            pytest.param(
+                *['noobaa_db'], marks=pytest.mark.polarion_id("OCS-2233")
+            )
+        ]
+    )
+    def test_restart_noobaa_resources(self, resource_to_delete):
+        """
+        Test Noobaa resources restart and check Noobaa health
```

Comment:
NooBaa resources are not restarted but deleted. After they are deleted the test waits for a new one to be created but it is not the original one so it is not restarted in my opinion.

Reply:
done